app/src/main/res/layout/recyclerview_block.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto">
+
+    <data>
+        <variable
+            name="block"
+            type="one.block.recenteosblocks.data.db.entities.Block" />
+
+    </data>
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <ImageView
+            android:id="@+id/eos_logo"
+            android:layout_width="100dp"
+            android:layout_height="100dp"
+            android:src="@drawable/eos"
+            app:layout_constraintTop_toTopOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            android:contentDescription="@string/app_name"/>
+        <LinearLayout
```

Comment:
Please don't chain layouts inside layouts, try to get this UI done with a single ConstraintLayout

Reply:
Done